index.html

```diff
@@ -4671,140 +4671,168 @@ <h2>Verification Method Rotation</h2>
+time the <a>verification method</a> was registered, to the time it was revoked.
       </p>
 
+
       <p>
-        If a verification method is no longer exclusively accessible to the
-        controller or parties trusted to act on behalf of the controller, it
-        is expected to be revoked immediately to reduce the risk of masquerading, theft,
-        and fraud.
+The following considerations might be of use when contemplating the use of
+<a>verification method</a> revocation:
       </p>
 
-      <p class="advisement">
+      <ul>
+        <li>
+<a>Verification method</a> revocation is a reactive security measure.
+        </li>
+
+        <li>
 It is considered a best practice to support key revocation.
-      </p>
+        </li>
 
-      <p class="advisement">
-A controller is expected to immediately revoke any verification method that is believed to
-be compromised.
-      </p>
+        <li>
+A <a>controller</a> is expected to immediately revoke any <a>verification
+method</a> that is believed to be compromised.
+        </li>
 
-      <p class="advisement">
-Revocation is expected to be understood as a controller expressing
-that proofs or signatures associated with a revoked verification method
-might have been created by an attacker. Verifiers, however, might
-still choose to accept or reject such proofs or signatures at their
-own discretion.
-      </p>
+        <li>
+<a>Verification method</a> revocation applies only to the latest version of a
+<a>DID Document</a>.
+        </li>
 
-      <p class="note">
-As described in <a href="#verification-material"></a>,
-absence of a verification method is the only form of revocation that
-applies to all DID Methods.
-      </p>
+        <li>
+As described in <a href="#verification-material"></a>, absence of a verification
+method is the only form of revocation that applies to all DID Methods.
+        </li>
 
-      <p>
-<a href="#method-operations"></a> specifies the <a>DID</a> operations to be
-supported by a <a>DID method</a> specification, including <a
-href="#method-operations">update</a> and <a href="#method-operations">
-deactivate</a> which might be used to remove verification method from a DID
-Document.
-      </p>
+        <li>
+If a <a>verification method</a> is no longer exclusively accessible to the
+<a>controller</a> or parties trusted to act on behalf of the <a>controller</a>,
+it is expected to be revoked immediately to reduce the risk of
+compromises such as masquerading, theft, and fraud.
+        </li>
 
-      <p>
-Not all DID Methods support verification method revocation.
-      </p>
+        <li>
+Revocation is expected to be understood as a <a>controller</a> expressing that
+proofs or signatures associated with a revoked <a>verification method</a> might
+have been created by an attacker. Verifiers, however, might still choose to
+accept or reject such proofs or signatures at their own discretion.
```

Comment:
```suggestion
Revocation is expected to be understood as a <a>controller</a> expressing that
proofs or signatures associated with a revoked <a>verification method</a>
created after its revocation should be treated as invalid. It could also imply
a concern that existing proofs or signatures may have been created by an
attacker, but this is not necessarily the case. Verifiers, however, might still
choose to accept or reject any such proofs or signatures at their own discretion.
```

Reply:
+1 for @dlongley 's suggestion.